app/web/components/UserSummary.tsx

```diff
@@ -77,25 +78,40 @@ export default function UserSummary({
     },
   );
 
+  const nameValue = user
+    ? nameOnly
+      ? user.name
+      : `${user.name}, ${user.age}`
+    : "";
+
```

Comment:
So when you add the age here, it means when it's past the limit, it won't show the age at all, like this:

<img width="412" alt="Screenshot 2025-05-12 at 7 49 29 PM" src="https://github.com/user-attachments/assets/b92813ce-7440-4d4d-bd95-38ddf8b0b7d4" />


What I'd suggest, is keep this variable for the nameValue only, then put the `nameOnly` logic back similar to how it was before, just using this variable instead, so:

`{nameOnly ? nameValue : `${nameValue}, ${user.age}`}`

Then this should be good to go!